src/components/ConditionalFilter/ValueProvider/UrlToken.ts

```diff
@@ -0,0 +1,47 @@
+// @ts-strict-ignore
+
+export const CONDITIONS = ["is", "equals", "in", "between", "lower", "greater"];
+
+const STATIC_TO_LOAD = ["category", "collection", "channel", "producttype"];
```

Comment:
```suggestion
const STATIC_TO_LOAD = ["category", "collection", "channel", "productType"];
```

Reply:
intentional, that name is uses everywhere in the API, keeping it that way we don't have to map all the time